oqs-template/generate.yml

```diff
@@ -10,6 +10,10 @@ kexs:
         name: 'nistp256'
         openssl_nid: 'NID_X9_62_prime256v1'
         pretty_name: 'ecdh-nistp256-frodokem-640-aesr2-sha256@openquantumsafe.org'
+      -
+        name: 'x25519'
+        x25519: true
```

Comment:
One of these two tags seems superfluous, no? Is there ever a config where "name: x25519" and "x25519: false"?

Reply:
See response in the overall PR thread.